inc/functions/functions-templates.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/**
+ * Display Authors
+ */
+function displayAuthors()
+{
+    global $post;
+
+    if (function_exists('get_post_meta')) {
+        $lead_author = get_post_meta($post->ID, 'lead_author', true);
+        $other_authors = get_post_meta($post->ID, 'other_author', true);
+
+        if (empty ($other_authors)) {
+            echo '<strong>Author:</strong>';
+        } else {
+            echo '<strong>Authors:</strong>';
+        }
+        echo ' ' . $lead_author;
+        if (!empty ($other_authors)) {
+            echo '; ' . $other_authors;
+        }
+    }
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displaySidebar()
+{
+    global $post;
+    if (function_exists('get_post_meta') & function_exists('get_sidebar')) {
+        if (get_post_meta($post->ID, 'sidebar', true) == 'false') {
+            // do nothing
+        } else {
+            get_sidebar(get_post_meta($post->ID, 'sidebar', true));
+        }
+    }
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displayDateOfPublication()
+{
+    global $post;
+
+    if (function_exists('get_post_meta')) {
+        echo date('d F Y', strtotime(get_post_meta($post->ID, 'date_published', true)));
+    }
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displayKeywordsTaxonomy()
+{
+    global $post;
+    if (function_exists('wp_get_post_terms')) {
+        //displaying custom taxonomy 'keywords'
+        $keywords_terms = wp_get_post_terms($post->ID, 'keywords');
+        $i = 0;
+        foreach ($keywords_terms as $term) {
+            $i++;
+            if ($i > 1) {
+                echo ', ';
+            }
+            echo $term->name;
+        }
+    }
+
+    return false;
+}
+
+/**
+ * @return bool
+ */
+function displayPublishedBy()
```

Comment:
Name convention should follow WP coding standards
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#naming-conventions